contracts/cw20-merkle-airdrop/src/state.rs

```diff
@@ -24,8 +26,17 @@ pub const STAGE_EXPIRATION: Map<u8, Expiration> = Map::new(STAGE_EXPIRATION_KEY)
 pub const STAGE_START_KEY: &str = "stage_start";
 pub const STAGE_START: Map<u8, Scheduled> = Map::new(STAGE_START_KEY);
 
+pub const STAGE_AMOUNT_KEY: &str = "stage_amount";
+pub const STAGE_AMOUNT: Map<u8, Uint128> = Map::new(STAGE_AMOUNT_KEY);
+
+pub const STAGE_AMOUNT_CLAIMED_KEY: &str = "claimed_amount";
+pub const STAGE_AMOUNT_CLAIMED: Map<u8, Uint128> = Map::new(STAGE_AMOUNT_CLAIMED_KEY);
+
 pub const MERKLE_ROOT_PREFIX: &str = "merkle_root";
 pub const MERKLE_ROOT: Map<u8, String> = Map::new(MERKLE_ROOT_PREFIX);
 
 pub const CLAIM_PREFIX: &str = "claim";
 pub const CLAIM: Map<(&Addr, u8), bool> = Map::new(CLAIM_PREFIX);
+
+pub const CLAIMED_AMOUNT_PREFIX: &str = "claimed_amount";
```

Comment:
Oops this is going to 🤯. Using same key for two states. Also naming is confusing.

Reply:
🤯 The tests were also passing anyways, very weird. Fixed and added more